src/hotspot/share/runtime/frame.inline.hpp

```diff
@@ -82,6 +83,7 @@ inline address frame::get_deopt_original_pc() const {
 }
 
 template <typename RegisterMapT>
+ATTRIBUTE_NO_UBSAN
```

Comment:
That's not good enough.  Turning off the ubsan warning doesn't prevent the compiler from doing
unexpected and potentially bad things with invalid code.